### PR TITLE
New Fruit: Pear

### PR DESCRIPTION
**Pull request by Adee07**
Added new Fruit. Must merge this to new branch: adeeBranch2

### CODE REVIEW

File: Fruits.txt [hunk 1]
```diff
@@ -6,4 +6,5 @@ Starfruit
 Jackfruits
 Watermelons
 Oranges
-Pineapple
\ No newline at end of file
+Pineapple
+Pear
```
Comment (Adee08) [inline Fruits.txt]: Will consider the Pineapple removal and addition for now. 
Approved.

Comment (Adee07) [reply Fruits.txt]: Thanks. Please approve and merge.

**Review by Adee08 — CHANGES_REQUESTED** [review]
Requesting change

**Review by Adee08 — APPROVED** [review]
Approving Changes

**Review by Adee08 — APPROVED** [review]
Approving again

**Review by Adee08 — APPROVED** [review]

**Pull request merged**